shard.yml

```diff
@@ -1,14 +1,17 @@
 name: json_rpc
 description: A JSON-RPC client and server library
-version: 0.1.0
+version: 0.1.1
+crystal: ">= 0.35.1, < 2.0.0"
+license: MIT
 
 authors:
   - Stefan Merettig <stefan-merettig@nuriaproject.org>
+  - Caspian Baska <caspianbaska@gmail.com>
 
 dependencies:
   cute:
     github: Papierkorb/cute
 
-crystal: 0.22.0
-
-license: MIT
+development_dependencies:
+  ameba:
```

Comment:
What's this for?

Reply:
This is a linter, [ameba](https://github.com/crystal-ameba/ameba). In pretty common use across crystal projects these days.

Reply:
Oh, Good that one standard has emerged beyond the built-in tool.